src/transaction.ts

```diff
@@ -93,6 +96,30 @@ export interface RunUpdateCallback {
   (err: null|ServiceError, rowCount: number): void;
 }
 
+export interface CommitRequest {
+  session: string;
+  transactionId?: Uint8Array|string|null;
+  singleUseTransaction?: ProtoITransactionOptions;
+  mutations: ProtoIMutation[];
+}
+
+export interface TransactionSelector {
+  singleUse?: ProtoITransactionOptions;
+  id?: Uint8Array|string|null;
+  begin?: ProtoITransactionOptions|null;
+}
+
+export type ProtoITransactionOptions = spanner_client.spanner.v1.ITransactionOptions;
+export type ProtoITransaction = spanner_client.spanner.v1.ITransaction;
+export type ProtoTransaction = spanner_client.spanner.v1.Transaction;
+export type ProtoITimestamp = spanner_client.protobuf.ITimestamp;
+export type ProtoBeginTransactionCallback = spanner_client.spanner.v1.Spanner.BeginTransactionCallback;
+export type ProtoIBeginTransactionRequest = spanner_client.spanner.v1.IBeginTransactionRequest;
+export type ProtoIReadOnly = spanner_client.spanner.v1.TransactionOptions.IReadOnly;
+export type ProtoCommitCallback = spanner_client.spanner.v1.Spanner.CommitCallback;
+export type ProtoCommitResponse = spanner_client.spanner.v1.CommitResponse;
+export type ProtoIMutation = spanner_client.spanner.v1.IMutation;
```

Comment:
Was wondering if the above should be put in `src/common.ts` to simplify type sourcing throughout the library. Optimistically speaking, we'd import the proto files once then distribute renamed types as needed from there

Reply:
I think that's ok? Although truthfully we shouldn't need most of these since the callbacks and grpc message classes aren't available through the client.

Reply:
I'll start clearing house then 👍